Cargo.toml

```diff
@@ -30,7 +30,7 @@ into_bits = []
 interpolate_idents = "0.2"
 arrayvec = { version = "^0.4", default-features = false }
 
-[target.'cfg(all(target_arch = "x86_64", target_feature = "sse2"))'.dependencies.sleef-sys]
```

Comment:
I think this can just be `target_arch = "x86_64"`. 32-bit targets do not work properly yet (sleef build system had some bugs, and sleef-sys is not updated yet).